### PR TITLE
FE-830 - Implement Hibana app styling controlled by theme context

### PR DESCRIPTION
**Pull request by nicklemmon**
[FE-830](https://jira.int.messagesystems.com/browse/FE-830)

**NOTE:** This is branched from `FE-904` - this should _not_ be merged before that one.

### What Changed

- Adds additional vertical padding to the navigation component to help offset the presence of the Hibana controls
- Incorporates styles from the UX mock ups
- Incorporates `<HibanaBanner/>` component in to the `<Layout/>` component. Child components control the rendering of a stylesheet and some descriptive content
- Reformats build and start Node scripts with Prettier
- Adds new `copyMatchboxStyles` Node script and incorporates in to the build and start scripts
- Updates tests to meet ticket requirements
- Updates the Node versions Travis uses to `10.17` to match AWS and our `"engines"` config in the `package.json`

### How To Test

1. Sign up for a new account
1. Update the UI flags via the `account/control` endpoint as described in #1443 
1. The banner should initially look like this
Screen Shot 2020-02-27 at 11 32 20 AM
1. Verify that on page load, a network request for `styles-default.css` is made
1. Click 'Take a Look' and the banner should look like this:
Screen Shot 2020-02-27 at 11 32 07 AM
1. Verify that a request was made for `styles-hibana.css` (you should see some visual changes in the app as well)
1. Additionally, verify that when all subsections within the navigation are expanded, that navigation items are not blocked by the Hibana banner - this should be checked on multiple viewports

### To Do
- [X] Get theme toggling working in the dev env
- [x] Restyle `<HibanaControls/>` to look like the mocks from a color/typography POV
- [x] Verify that styled system props work
- [x] Set up production build webpack config stuff
- [x] Ensure both CSS files are _not_ requested unless the user swaps to the new design
- [x] Verify that the CSS files are overwritten on each build
- [x] Travis builds are failing due to invalid `package-lock.json`

---

- [ ] Incorporate feedback

### CODE REVIEW

**Comment by PatrickSison** [comment]
Issue with the navbar going behind banner is still present in Firefox, at least for me. Good in chrome.

Screen Shot 2020-03-02 at 9 12 46 AM

**Comment by nicklemmon** [comment]
> Issue with the navbar going behind banner is still present in Firefox, at least for me. Good in chrome.
> 
> Screen Shot 2020-03-02 at 9 12 46 AM

Good catch! I'll check it out.

File: cypress/tests/integration/accounts/changeBillingPlanPage.spec.js [hunk 1]
```diff
@@ -101,19 +99,25 @@ describe('Billing Page', () => {
...
+
     cy.stubRequest({
       url: '/api/v1/billing/subscription',
       fixture: 'billing/subscription/200.get.test-plan.json',
+      requestAlias: 'subGet',
     });
+
     cy.visit('/account/billing/plan?code=50K-starter-0519');
 
+    cy.wait(['@accountGet', '@billingGet', '@subGet']);
```
[the remark this answers is not in the document]
Comment (nicklemmon) [reply cypress/tests/integration/accounts/changeBillingPlanPage.spec.js]: I was seeing some flake on this one, and this "resolved" it, but who knows if it actually did without the video evidence...

**Comment by nicklemmon** [comment]
> Looks good. The banner is a bit large for mobile view ports, so not sure if we want to fix now or leave it for later on. Either hiding it or making it so users can dismiss the banner.

I shared your concern with UX - Aubrey is going to think about it a bit and we may make a tweak before merging.

**Pull request merged**